src/components/cards/mealComponent/PEMealComponentTouch.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import { type ReactElement } from 'react';
+import { Icon } from '../../standard/icon/Icon';
+import PeopleEatIcon from '../../standard/icon/PEIcon';
+
+export interface PEMealComponentProps {
+    onClick?: () => void;
+    mealTitle?: string;
+    mealDescription?: string;
+    mealPicture?: string;
+}
+
+export default function PEMealComponentTouch({ onClick, mealTitle, mealDescription, mealPicture }: PEMealComponentProps): ReactElement {
+    return (
+        <div
+            onClick={(): void => onClick?.()}
```

Comment:
Wouldn't this work just as well?

```suggestion
            onClick={onClick}
```